### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kwinborne**
Created a skeleton for further development of documentation. Added "How to help" and table of contents.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,5 +32,15 @@
    ### <ins>Changed:</ins>
    ```diff
    ! [sambutler1017] - CodeOwners to include @llengel and @kwinborne
-   ! [kwinborne] - Merged UserProfile and UserCredentials into User
+   ! [kwinborne] - Merged UserProfile and UserCredentials into Use
+   ```
+## Release - 07/28/2020]
+   ### <ins>Added:</ins>
```
Comment (sambutler1017) [inline CHANGELOG.md:38]: You don't need to put the added tag if there is nothing being added

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,5 +32,15 @@
    ### <ins>Changed:</ins>
    ```diff
    ! [sambutler1017] - CodeOwners to include @llengel and @kwinborne
-   ! [kwinborne] - Merged UserProfile and UserCredentials into User
+   ! [kwinborne] - Merged UserProfile and UserCredentials into Use
+   ```
+## Release - 07/28/2020]
```
Comment (sambutler1017) [inline CHANGELOG.md:37]: This date is wrong and there is an unneeded bracket

File: CHANGELOG.md (hunk 1 above)
Comment (kwinborne) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (kwinborne) [reply CHANGELOG.md]: fixed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -32,5 +32,10 @@
    ### <ins>Changed:</ins>
    ```diff
    ! [sambutler1017] - CodeOwners to include @llengel and @kwinborne
-   ! [kwinborne] - Merged UserProfile and UserCredentials into User
+   ! [kwinborne] - Merged UserProfile and UserCredentials into Use
+   ```
+## Release - 07/27/2020
+   ### <ins>Changed:</ins>
+   ```diff
+   ! Updated Readme.md to include the skeleton for a guide for beginners
```
Comment (sambutler1017) [inline CHANGELOG.md:40]: Put your name in in brackets `[kwinborne]`

Comment (kwinborne) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -32,5 +32,10 @@
    ### <ins>Changed:</ins>
    ```diff
    ! [sambutler1017] - CodeOwners to include @llengel and @kwinborne
-   ! [kwinborne] - Merged UserProfile and UserCredentials into User
+   ! [kwinborne] - Merged UserProfile and UserCredentials into Use
+   ```
+## Release - 07/27/2020
+   ### <ins>Changed:</ins>
+   ```diff
+   ! [kwinborne] Updated Readme.md to include the skeleton for a guide for beginners
```
Comment (sambutler1017) [inline CHANGELOG.md:40]: With a dash lol

Comment (kwinborne) [reply CHANGELOG.md:40]: oops

**Pull request merged**